vite-project/src/App.tsx

```diff
@@ -15,11 +44,33 @@ const App: React.FC = () => {
     setProjects(projects.filter((project) => project.id !== id));
   };
 
+  const toggleDarkMode = () => {
+    setDarkMode((prevDarkMode) => !prevDarkMode);
+  };
+
+  const theme = darkMode ? darkTheme : lightTheme;
+
   return (
-    <div className="main">
-      <AddOrderForm addProject={addProject} />
-      <Cards projects={projects} deleteProject={deleteProject} />
-    </div>
+    <ThemeProvider theme={theme}>
+      <CssBaseline />
+      <div
+        className={`main ${darkMode ? "dark-background" : "light-background"}`}
+      >
+        <IconButton
+          onClick={toggleDarkMode}
+          aria-label="toggle dark mode"
+          sx={{ position: "absolute", top: 10, right: 10 }}
```

Comment:
Just for the future, when we combine your projects this week, let's call at the stage of writing styles and work together on responsiveness, I'll give you some tips